app.py

```diff
@@ -10,13 +11,55 @@
 @app.route('/get_weather', methods=['POST'])
 def get_weather():
     data = request.get_json()
+    #Use Today's date if date isn't given
+    if 'date' not in data or not data['date']:
+        data['date'] = datetime.now().strftime('%Y-%m-%d')
     request_model = ForecastRequestModel(**data) #have to unpack data apparently for pydantic model
     #except ValidationError as e:
     #   return jsonify(e.errors()), 400
     
     response = forecast_service.get_forecast(request_model)
     return jsonify(response.dict()), response.status_code
 
+@app.route('/get_7_day_forecast', methods=['POST'])
+def get_7_day_forecast():
+    data = request.get_json()
+
+    if 'date' not in data or not data['date']:
+        data['date'] = datetime.now().strftime('%Y-%m-%d')
+    
+    request_model = ForecastRequestModel(**data) 
+    #except ValidationError as e:
+    #   return jsonify(e.errors()), 400
+    response = forecast_service.get_7_day_forecast(request_model)
+    return jsonify(response.dict()), response.status_code
+
+@app.route('/get_current_forecast', methods=['POST'])

```

Comment:
For this route the datetime should always be now so would we need to pass anything here besides zipcode? Could we change this then to be a GET method that has the zipcode as a path parameter like /forecast/<zipcode> or /forecast/current/<zipcode>